### DISCUSSION TITLE
Sentinel添加流控、降级、热点、授权，成功后，但是页面没有显示记录，也不生效啊

### DISCUSSION THREAD

**Discussion by 2366095032** [alibaba/Sentinel#2203] [Q&A]
进入sentinel页面，簇点链路可以显示，流量监控也是正常，### 给刚刚访问的请求添加流控，但是成功后没有显示记录，也没有生效，1.7,1.8都用过了，咋办呢，项目控制台没有报错，求解答

**Comment by sahnggq** [comment]
我也出现这个情况，是跟dubbo进行连接的时候，流控规则在添加的时候，页面没有任何反应，后台也没有出错，现在也没有解决
